protonvpn_cli/utils.py

```diff
@@ -475,3 +476,38 @@ def is_valid_ip(ipaddr):
 
     else:
         return False
+
+
+def get_transferred_data():
+    """Reads and returns the amount of data transferred during a session
+    from the /sys/ directory"""
+
+    def convert_size(size_bytes):
+        """Converts byte amounts into human readable formats"""
+        if size_bytes == 0:
+            return "0B"
+        size_name = ("B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB")
+
+        i = int(math.floor(math.log(size_bytes, 1000)))
+        p = math.pow(1000, i)
+        s = round(size_bytes / p, 2)
+        return "{0} {1}".format(s, size_name[i])
```

Comment:
What about using: https://pypi.org/project/humanfriendly/ ?

Reply:
I don't think it's great to introduce another external dependency for such a simple task.

Only thing I know is that the current way would break if someone would surpass 999 YB (list index out of range), but I guess that's an acceptable risk, no? :P